src/Contrib/Grpc/GrpcTransportFactory.php

```diff
@@ -20,8 +20,16 @@
 
 final class GrpcTransportFactory implements TransportFactoryInterface
 {
+    /**
+     * @psalm-param "application/x-protobuf" $contentType
```

Comment:
Didn't find a solution that works with template type on `TransportFactoryInterface`.